src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/webspace/settings/snippets/main.js

```diff
@@ -13,8 +13,8 @@ define(['config', 'app-config'], function(Config, AppConfig) {
 
     var defaults = {
         options: {
-            snippetTypesUrl: '/admin/api/snippet-types?defaults=true&webspace=<%= webspace %>',
-            snippetTypeDefaultUrl: '/admin/api/snippet-types/<%= type %>/default?webspace=<%= webspace %>',
+            snippetAreasUrl: '/admin/api/snippet-types/areas?webspace=<%= webspace %>',
```

Comment:
In theory it is more than an internal BC break (it's a REST API accessible by everybody...). Would it be an option to leave these options in the old controller for now, and add a comment there and in the new Controller?